### DISCUSSION TITLE
To Do List

### DISCUSSION THREAD

**Discussion by elParaguayo** [qtile/qtile#4033] [General]
We were wondering if it would be useful  to keep a list of the various things that we're working on so there's some visibility about what we're doing. See below for an example.

My main concern with this is that we may simply forget to keep updating it but interested to know what people think would be useful.

### Documentation
| Item | Assignee | Status |
|----|----|-----|
| Document all available commands | @elParaguayo | In progress |
| Previews for layouts | @elParaguayo | Started |

### Widgets
| Item | Assignee | Status |
|----|----|-----|
| Update volume widget | @adolfgatonegro | PR submitted #3332 (but also #3757) |
| Allow user specificed precision for `Net` widget | Various | #3487 #4023 |
| Add mute to `PulseVolume` | @m-col | #3972 |
| Better metdata handling in `Mpris2` | @elParaguayo | #3904 |
| Add SVG support to `CurrentLayoutIcon` | @elParaguayo | #4008 |

### Layouts
| Item | Assignee | Status |
|----|----|-----|
| Add `grow_left/right` commands to Monad layouts | @martanman | #4001 |
| Add single border width to Columns and VerticalTile layouts | @matthewsia98 | #3990 #3991 (but also #3711) |
| Add new `ScreenSplit` layout | @elParaguayo | #3793 |

### Wayland
| Item | Assignee | Status |
|----|----|-----|

### X11
| Item | Assignee | Status |
|----|----|-----|

### Qtile core
| Item | Assignee | Status |
|----|----|-----|
| HiDPI support | | |
| Lag with high mouse polling rate (#3680) | | |

**Comment by m-col** [comment]
Would it be easier for maintainability if we used one of the tables interfaces that github provides? For example see my wayland tickets table: https://github.com/orgs/qtile/projects/2/views/2. Much like as you've done above, we can define arbitrary tags, assignees, link PRs/issues (and see there when they get updated), add notes etc
